lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -1013,8 +1015,10 @@ def set_required_metadata_variables
       # All metadata parameters must now be set.
       missing = []
       missing << 'project_id' unless @project_id
-      missing << 'zone' unless @zone
-      missing << 'vm_id' unless @vm_id
+      if @platform != Platform::OTHER
+        missing << 'zone' unless @zone
```

Comment:
Doesn't this go away altogether in #241?

Reply:
Yes, but for now Ling has asked that we keep it, and I agree with this for doing the minimally disruptive action until we're ready to follow through with a larger refactoring.